### PR TITLE
Associate report to node

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 1]
```diff
@@ -384,8 +383,8 @@ public Mono<Void> deleteModificationGroup(UUID groupUuid) {
 
     public List<ModificationInfos> doAction(NetworkStoreListener listener, Runnable action,
                                             NetworkModificationException.Type typeIfError,
-                                            UUID networkUuid, ReporterModel reporter,
-                                            Reporter subReporter) {
+                                            UUID reportUuid, Supplier<ReporterModel> reporter,
```
Comment (jonenst) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:386]: why change from reporter to Supplier<Reporter> ?

Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Removed

Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Delete also for subReporter ?

Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Only do AtomicReference.get

Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Done

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**